tools/cli/domainCommands.go

```diff
@@ -282,6 +282,28 @@ func (d *domainCLIImpl) UpdateDomain(c *cli.Context) {
 	}
 }
 
+func (d *domainCLIImpl) DeprecateDomain(c *cli.Context) {
+	domainName := getRequiredGlobalOption(c, FlagDomain)
+	securityToken := c.String(FlagSecurityToken)
+
+	ctx, cancel := newContext(c)
+	defer cancel()
+
+	err := d.deprecateDomain(ctx, &types.DeprecateDomainRequest{
+		Name: domainName,
+		SecurityToken: securityToken,
+	})
+	if err != nil {
+		if _, ok := err.(*s.EntityNotExistsError); !ok {
```

Comment:
As call uses types from `types` package, you should also check agains an error (with same name) from this package.
This was probably copy-paste, in that case could you check if there are any more cases like this?